Makefile

```diff
@@ -296,3 +296,10 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+# Build and run lvm tests.
+E2E:
+	@echo "build and run e2e tests"
+	cd e2e/lvm && ginkgo build
+	cd e2e/lvm && ./lvm.test --lvm-catalog-image=quay.io/rsinghal/lvm-operator-catalog --lvm-subscription-channel="alpha"
```

Comment:
Please use the CATALOG_IMG variable here instead of a hardcoded path. Also do the same for channel.

Reply:
Updated